.travis.yml

```diff
@@ -2,6 +2,9 @@ language: python
 
 before_script:
 - pip install tox
+- sed -i 's/\/opt/\/home\/travis\/build\/rcbops/g' ansible-role-requirements.yml
+- pip2 install --force-reinstall 'ansible===1.9.4'
+- ansible-galaxy install --role-file=ansible-role-requirements.yml --ignore-errors --force
```

Comment:
there is even a bug about it being an overloaded option https://github.com/ansible/ansible/issues/9744